## 84% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 7
Expected to fail tests: 1
Passed tests: 43
```
```
IDs of failed tests: 727230,704613,703495,703503,727229,702775,703629 
```
```
IDs of expected to fail tests: 727232 
```
<h3>Failed tests (7)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=20463-collectible-cant-fetch-info-iOS&PR_ID=20714&APK_NAME=20714.apk&TR_CASE_IDS=727230,704613,703495,703503,727229,702775,703629">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605821">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping up </div><div># STEP: Check Status Inu community tags in the Discover communities screen</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:299: in test_community_discovery
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community ' 0xUX' is not in the Discover Communities list.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/880391e36abb4c44bdd9edc9489702ef?auth=ed084749851b59271032d1ff376a8562#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605826">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `top-bar` </div><div>Device 1: Find `Button` by `accessibility id`: `top-bar` </div></blockquote></p>

```
critical/test_wallet.py:173: in test_wallet_send_asset_from_drawer
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.0 but expected to be -0.0001
E    Receiver balance is not updated on Etherscan, it is 0.0026 but expected to be 0.0027
E    Eth amount in the receivers wallet is 0.016 but should be 0.0161
E    Eth amount in the senders wallet is 0.1797 but should be 0.1796
E    The last transaction is not listed in activity for the receiver, expected timestamp is Today 6:55 PM
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605825">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `top-bar` </div><div>Device 2: Find `Button` by `accessibility id`: `top-bar` </div></blockquote></p>

```
critical/test_wallet.py:147: in test_wallet_send_eth
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.0 but expected to be -0.0001
E    Receiver balance is not updated on Etherscan, it is 0.0026 but expected to be 0.0027
E    Eth amount in the receivers wallet is 0.016 but should be 0.0161
E    Eth amount in the senders wallet is 0.1797 but should be 0.1796
E    The last transaction is not listed in activity for the receiver, expected timestamp is Today 6:51 PM
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605822">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for community: 'open community' </div><div>Device 2: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1207: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Joined status is not displayed
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/542bfa622bbb4b7089b8ccf7156a03df?auth=e707d543bd3f9417647bdfab68fa20f1#494">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a9f7679aaa44dc79075b10ac97aafbc?auth=3ef67f8f213abeea7e11cf9ab29b3107#478">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605823">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for chat: 'general' </div><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:71: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'token-gated community e2e' was not requested to join by the url https://status.app/c/G0IAAGS9TbI9SrSPbDPplfBUeBBOIWhFRhGIKVlxntAg5CSvsJw4jurb636UKo7Lm2gTUS_M#zQ3shm7cS2MX6K1iinjNhiw88g1WH6fbE8rvnMHBq4RhznkWc
E    Community 'open community' was not requested to join by the url https://status.app/c/G0UAAMTyNsn2QZDEG0EXftOl8pOEfwEBOOSA_YTfIk85xmADDgINGmxpUHAXzK36bN0fK42Xf4YD2yjPk1z2pbFwFw==#zQ3shZU9PNP6QHwEmqoh5mQqjCMDsfZYnpY4BEC8hom7KoNEz
E    Community 'closed community' was not requested to join by the url https://status.app/c/G00AAGS9TbI9mSR-ZNmFrhRjNuEeXAAbcAIUaLLJyjMOG3ACJQ12oIHD78QhzO9s_T5bUeU7rnATWJg3mGgTUemrAg==#zQ3shTK1zXmLq2yZWMij65j1LNtmvuG6x71Nmg2fwtKs32wHj
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c234cec2c8cd484da1bae2968a0a4425?auth=5548c831ea8bd387ee93fe0d0e838e4c#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605784">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `BrowserTab` by `accessibility id`: `browser-stack-tab` </div><div>Device 1: Tap on found: BrowserTab </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:115: in test_links_deep_links
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'token-gated community e2e' was not requested to join by the deep link status.app://c/G0IAAGS9TbI9SrSPbDPplfBUeBBOIWhFRhGIKVlxntAg5CSvsJw4jurb636UKo7Lm2gTUS_M#zQ3shm7cS2MX6K1iinjNhiw88g1WH6fbE8rvnMHBq4RhznkWc
E    Community 'open community' was not requested to join by the deep link status.app://c/G0UAAMTyNsn2QZDEG0EXftOl8pOEfwEBOOSA_YTfIk85xmADDgINGmxpUHAXzK36bN0fK42Xf4YD2yjPk1z2pbFwFw==#zQ3shZU9PNP6QHwEmqoh5mQqjCMDsfZYnpY4BEC8hom7KoNEz
E    Community 'closed community' was not requested to join by the deep link status.app://c/G00AAGS9TbI9mSR-ZNmFrhRjNuEeXAAbcAIUaLLJyjMOG3ACJQ12oIHD78QhzO9s_T5bUeU7rnATWJg3mGgTUemrAg==#zQ3shTK1zXmLq2yZWMij65j1LNtmvuG6x71Nmg2fwtKs32wHj
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c234cec2c8cd484da1bae2968a0a4425?auth=5548c831ea8bd387ee93fe0d0e838e4c#214">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605819">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div>Device 2: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div><div>Device 2: Looking for a message by text: Chat is unmuted now </div></blockquote></p>

```
critical/chats/test_group_chat.py:520: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Text 'Muted until 02:51 today' is not shown for muted chat
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7597fea7ec944b80b4fe0361d4af9d24?auth=1142a6a16e177f60a255173d9b0a1eca#475">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab6918c9b08b46afa508fdb536f43a0c?auth=a578b46ab1cb7839ce47df2ccd8ecbb0#273">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8dc44800ccd48b2b0acf6267234c2d7?auth=4f5f8efc4bdc6aedd11d581eb04f84f3#244">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605828">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EditBox` by `accessibility id`: `add-address-to-watch` </div><div>Device 1: Type `0x8d2413447ff297d30bdc475f6d5cb00254685aae` to `EditBox` </div></blockquote></p>

```
critical/test_wallet.py:237: in test_wallet_add_remove_watch_only_account
    self.wallet_view.add_watch_only_account(address=address_to_watch, account_name=new_account_name)
../views/wallet_view.py:150: in add_watch_only_account
    self.account_has_activity_label.wait_for_visibility_of_element()
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: Text by accessibility id:`account-has-activity` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/08d04003bcf844f5a9b96f6d82fca49b?auth=c60fd404caa51e158e4a624aea498b65#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (43)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605785">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7c0483d10b40fa82a8cbdee6b9a471?auth=5e73e3c4f372852776f19656247e1b1b#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d381126eb5b4c9da9233b0bc72b1497?auth=2fae52ac0e697948f9dc270031ff419b#210">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605803">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7c0483d10b40fa82a8cbdee6b9a471?auth=5e73e3c4f372852776f19656247e1b1b#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d381126eb5b4c9da9233b0bc72b1497?auth=2fae52ac0e697948f9dc270031ff419b#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605802">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7c0483d10b40fa82a8cbdee6b9a471?auth=5e73e3c4f372852776f19656247e1b1b#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d381126eb5b4c9da9233b0bc72b1497?auth=2fae52ac0e697948f9dc270031ff419b#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605808">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c969b8265b37466486e6c530fcc13965?auth=a16f4e96996c41229e9eb2eb48623081#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bdaeb25220e94ad59802c67bc989dfe9?auth=c264021060d3ed9692f3b92488acbb6e#98">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605813">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/880391e36abb4c44bdd9edc9489702ef?auth=ed084749851b59271032d1ff376a8562#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605782">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/880391e36abb4c44bdd9edc9489702ef?auth=ed084749851b59271032d1ff376a8562#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605806">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/880391e36abb4c44bdd9edc9489702ef?auth=ed084749851b59271032d1ff376a8562#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605801">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/880391e36abb4c44bdd9edc9489702ef?auth=ed084749851b59271032d1ff376a8562#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605817">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/880391e36abb4c44bdd9edc9489702ef?auth=ed084749851b59271032d1ff376a8562#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605810">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7fb63160d9a644ae8fb513f91972649b?auth=7a08e6a1415d36a66c2aeeaac750c89a#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/587249478a324b23aa9ee197b284902c?auth=c8c0e886a2a8fa2c909707be70e238ce#100">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605811">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7fb63160d9a644ae8fb513f91972649b?auth=7a08e6a1415d36a66c2aeeaac750c89a#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/587249478a324b23aa9ee197b284902c?auth=c8c0e886a2a8fa2c909707be70e238ce#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605788">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a76bdf0cc39f462ea82dfd9eb0d03978?auth=db8dc177febb38c9487f74e460a1ab23#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a47cd813b234815a36682816cbb35b3?auth=bd3838785b1e517e6f2bbcfb3bc1bfb0#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605787">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a76bdf0cc39f462ea82dfd9eb0d03978?auth=db8dc177febb38c9487f74e460a1ab23#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a47cd813b234815a36682816cbb35b3?auth=bd3838785b1e517e6f2bbcfb3bc1bfb0#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605820">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a76bdf0cc39f462ea82dfd9eb0d03978?auth=db8dc177febb38c9487f74e460a1ab23#105">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5a47cd813b234815a36682816cbb35b3?auth=bd3838785b1e517e6f2bbcfb3bc1bfb0#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605792">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/542bfa622bbb4b7089b8ccf7156a03df?auth=e707d543bd3f9417647bdfab68fa20f1#180">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a9f7679aaa44dc79075b10ac97aafbc?auth=3ef67f8f213abeea7e11cf9ab29b3107#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605809">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/542bfa622bbb4b7089b8ccf7156a03df?auth=e707d543bd3f9417647bdfab68fa20f1#299">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a9f7679aaa44dc79075b10ac97aafbc?auth=3ef67f8f213abeea7e11cf9ab29b3107#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605789">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/542bfa622bbb4b7089b8ccf7156a03df?auth=e707d543bd3f9417647bdfab68fa20f1#131">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a9f7679aaa44dc79075b10ac97aafbc?auth=3ef67f8f213abeea7e11cf9ab29b3107#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605800">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/542bfa622bbb4b7089b8ccf7156a03df?auth=e707d543bd3f9417647bdfab68fa20f1#296">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a9f7679aaa44dc79075b10ac97aafbc?auth=3ef67f8f213abeea7e11cf9ab29b3107#269">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605827">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/08d04003bcf844f5a9b96f6d82fca49b?auth=c60fd404caa51e158e4a624aea498b65#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605786">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9a79ddcec6a4325a3bc4de2e3230fb9?auth=97894e1d6472340e6e72b4eb6faa0050">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9bfcc50b809449a889dba329ad9ca5ff?auth=e4a854180c7ca785dc01576a2db14108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605781">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9a79ddcec6a4325a3bc4de2e3230fb9?auth=97894e1d6472340e6e72b4eb6faa0050">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9bfcc50b809449a889dba329ad9ca5ff?auth=e4a854180c7ca785dc01576a2db14108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605793">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9a79ddcec6a4325a3bc4de2e3230fb9?auth=97894e1d6472340e6e72b4eb6faa0050">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9bfcc50b809449a889dba329ad9ca5ff?auth=e4a854180c7ca785dc01576a2db14108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605783">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9a79ddcec6a4325a3bc4de2e3230fb9?auth=97894e1d6472340e6e72b4eb6faa0050">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9bfcc50b809449a889dba329ad9ca5ff?auth=e4a854180c7ca785dc01576a2db14108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605804">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9a79ddcec6a4325a3bc4de2e3230fb9?auth=97894e1d6472340e6e72b4eb6faa0050">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9bfcc50b809449a889dba329ad9ca5ff?auth=e4a854180c7ca785dc01576a2db14108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605818">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9a79ddcec6a4325a3bc4de2e3230fb9?auth=97894e1d6472340e6e72b4eb6faa0050">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9bfcc50b809449a889dba329ad9ca5ff?auth=e4a854180c7ca785dc01576a2db14108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605779">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9a79ddcec6a4325a3bc4de2e3230fb9?auth=97894e1d6472340e6e72b4eb6faa0050">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9bfcc50b809449a889dba329ad9ca5ff?auth=e4a854180c7ca785dc01576a2db14108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605778">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9a79ddcec6a4325a3bc4de2e3230fb9?auth=97894e1d6472340e6e72b4eb6faa0050">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9bfcc50b809449a889dba329ad9ca5ff?auth=e4a854180c7ca785dc01576a2db14108">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605780">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7597fea7ec944b80b4fe0361d4af9d24?auth=1142a6a16e177f60a255173d9b0a1eca#355">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab6918c9b08b46afa508fdb536f43a0c?auth=a578b46ab1cb7839ce47df2ccd8ecbb0#230">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8dc44800ccd48b2b0acf6267234c2d7?auth=4f5f8efc4bdc6aedd11d581eb04f84f3#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605816">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7597fea7ec944b80b4fe0361d4af9d24?auth=1142a6a16e177f60a255173d9b0a1eca#282">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab6918c9b08b46afa508fdb536f43a0c?auth=a578b46ab1cb7839ce47df2ccd8ecbb0#168">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8dc44800ccd48b2b0acf6267234c2d7?auth=4f5f8efc4bdc6aedd11d581eb04f84f3#143">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605815">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7597fea7ec944b80b4fe0361d4af9d24?auth=1142a6a16e177f60a255173d9b0a1eca#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab6918c9b08b46afa508fdb536f43a0c?auth=a578b46ab1cb7839ce47df2ccd8ecbb0#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8dc44800ccd48b2b0acf6267234c2d7?auth=4f5f8efc4bdc6aedd11d581eb04f84f3#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605790">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7597fea7ec944b80b4fe0361d4af9d24?auth=1142a6a16e177f60a255173d9b0a1eca#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab6918c9b08b46afa508fdb536f43a0c?auth=a578b46ab1cb7839ce47df2ccd8ecbb0#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8dc44800ccd48b2b0acf6267234c2d7?auth=4f5f8efc4bdc6aedd11d581eb04f84f3#80">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605791">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7597fea7ec944b80b4fe0361d4af9d24?auth=1142a6a16e177f60a255173d9b0a1eca#322">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab6918c9b08b46afa508fdb536f43a0c?auth=a578b46ab1cb7839ce47df2ccd8ecbb0#203">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8dc44800ccd48b2b0acf6267234c2d7?auth=4f5f8efc4bdc6aedd11d581eb04f84f3#203">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605814">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#229">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605805">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#254">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#227">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605796">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#302">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#287">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605812">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#548">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#473">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605807">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#421">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#397">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605824">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#583">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#499">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605795">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#200">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#135">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605794">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605799">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#326">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#306">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605798">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#176">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3605797">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5cfc7da0d87640289dc663d986380507?auth=bf10d01a7a9648da7bcf713dec1cb078#391">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5232b183d954f5d8e9de0c1d8b9866a?auth=f4029d7049a4c31d1af5b5e514c60f85#367">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/quo/components/profile/expanded_collectible/view.cljs
+++ b/src/quo/components/profile/expanded_collectible/view.cljs
@@ -79,14 +79,20 @@
         [counter-view counter])
       [rn/view {:style (style/collectible-border theme)}]]]))
 
+(defn invalid-image?
+  [image-src]
+  (or (nil? image-src)
+      (string/blank? image-src)))
+
 (defn view-internal
   [{:keys [container-style square? on-press counter image-src native-ID supported-file?
            on-collectible-load aspect-ratio gradient-color-index]
     :or   {gradient-color-index :gradient-1
            on-collectible-load  (fn [])}}]
   (let [theme              (quo.theme/use-theme)
-        [error? set-error] (rn/use-state (or (nil? image-src)
-                                             (string/blank? image-src)))]
+        [error? set-error] (rn/use-state (invalid-image? image-src))]
+    (rn/use-effect #(set-error (invalid-image? image-src))
+                   [image-src])
     [rn/pressable
      {:style               (merge container-style (style/container aspect-ratio))
       :accessibility-label :expanded-collectible

--- a/src/status_im/contexts/wallet/collectible/events.cljs
+++ b/src/status_im/contexts/wallet/collectible/events.cljs
@@ -185,7 +185,7 @@
            ;; We delay the navigation because we need re-frame to update the DB on time.
            ;; By doing it, we skip a blink while visiting the collectible detail page.
            [:dispatch-later
-            {:ms       1
+            {:ms       17
              :dispatch [:navigate-to :screen/wallet.collectible]}]]})))
 
 (defn- keep-not-empty-value

--- a/src/status_im/contexts/wallet/collectible/view.cljs
+++ b/src/status_im/contexts/wallet/collectible/view.cljs
@@ -79,12 +79,20 @@
   (rf/dispatch [:wallet/clear-collectible-details]))
 
 (defn animated-header
-  [{:keys [scroll-amount title-opacity page-nav-type picture title description theme id]}]
-  (let [blur-amount   (header-animations/use-blur-amount scroll-amount)
-        layer-opacity (header-animations/use-layer-opacity
-                       scroll-amount
-                       (colors/theme-colors colors/white-opa-0 colors/neutral-95-opa-0 theme)
-                       (colors/theme-colors colors/white-opa-50 colors/neutral-95-opa-70-blur theme))]
+  [{:keys [scroll-amount title-opacity page-nav-type theme]}]
+  (let [blur-amount               (header-animations/use-blur-amount scroll-amount)
+        layer-opacity             (header-animations/use-layer-opacity
+                                   scroll-amount
+                                   (colors/theme-colors colors/white-opa-0
+                                                        colors/neutral-95-opa-0
+                                                        theme)
+                                   (colors/theme-colors colors/white-opa-50
+                                                        colors/neutral-95-opa-70-blur
+                                                        theme))
+        {{preview-uri :uri}  :preview-url
+         {title :name}       :collectible-data
+         {description :name} :collection-data
+         id                  :id} (rf/sub [:wallet/collectible-details])]
     [rn/view {:style (style/animated-header)}
      [reanimated/blur-view
       {:style         {:flex             1
@@ -97,7 +105,7 @@
       [reanimated/view {:style layer-opacity}
        [quo/page-nav
         {:type                page-nav-type
-         :picture             picture
+         :picture             preview-uri
          :title               title
          :description         description
          :background          :blur
@@ -110,7 +118,7 @@
                                               {:content (fn []
                                                           [options-drawer/view
                                                            {:name  title
-                                                            :image picture
+                                                            :image preview-uri
                                                             :id    id}])
                                                :theme   theme}])}]
          :center-opacity      title-opacity}]]]]))
@@ -144,10 +152,11 @@
   [_]
   (let [selected-tab  (reagent/atom :overview)
         on-tab-change #(reset! selected-tab %)]
-    (fn [{:keys [collectible set-title-bottom theme]}]
+    (fn [{:keys [set-title-bottom theme]}]
       (let [title-ref                      (rn/use-ref-atom nil)
             set-title-ref                  (rn/use-callback #(reset! title-ref %))
             animation-shared-element-id    (rf/sub [:animation-shared-element-id])
+            collectible                    (rf/sub [:wallet/collectible-details])
             collectible-owner              (rf/sub [:wallet/collectible-details-owner collectible])
             aspect-ratio                   (rf/sub [:wallet/collectible-aspect-ratio])
             gradient-color                 (rf/sub [:wallet/collectible-gradient-color])
@@ -196,14 +205,14 @@
                                         token-id
                                         {:images           [collectible-image]
                                          :index            0
-                                         :on-options-press #(rf/dispatch [:show-bottom-sheet
-                                                                          {:content
-                                                                           (fn []
-                                                                             [options-drawer/view
-                                                                              {:name collectible-name
-                                                                               :image
-                                                                               preview-uri
-                                                                               :id id}])}])}])))
+                                         :on-options-press #(rf/dispatch
+                                                             [:show-bottom-sheet
+                                                              {:content
+                                                               (fn []
+                                                                 [options-drawer/view
+                                                                  {:name  collectible-name
+                                                                   :image preview-uri
+                                                                   :id    id}])}])}])))
             :on-collectible-load  (fn []
                                     ;; We need to delay the measurement because the
                                     ;; navigation has an animation
@@ -241,33 +250,24 @@
 
 (defn view
   [_]
-  (let [{:keys [top]}       (safe-area/get-insets)
-        theme               (quo.theme/use-theme)
-        title-bottom-coord  (rn/use-ref-atom 0)
-        set-title-bottom    (rn/use-callback
-                             (fn [_ y _ height]
-                               (reset! title-bottom-coord
-                                 (get-title-bottom-y-position y height))))
-        scroll-amount       (reanimated/use-shared-value 0)
-        title-opacity       (reanimated/use-shared-value 0)
-        {:keys [collection-data
-                collectible-data
-                preview-url]
-         :as   collectible} (rf/sub [:wallet/collectible-details])]
+  (let [{:keys [top]}      (safe-area/get-insets)
+        theme              (quo.theme/use-theme)
+        title-bottom-coord (rn/use-ref-atom 0)
+        set-title-bottom   (rn/use-callback
+                            (fn [_ y _ height]
+                              (reset! title-bottom-coord
+                                (get-title-bottom-y-position y height))))
+        scroll-amount      (reanimated/use-shared-value 0)
+        title-opacity      (reanimated/use-shared-value 0)]
     [rn/view {:style (style/background-color theme)}
      [animated-header
-      {:id            (:id collectible)
-       :scroll-amount scroll-amount
+      {:scroll-amount scroll-amount
        :title-opacity title-opacity
        :page-nav-type :title-description
-       :picture       (:uri preview-url)
-       :title         (:name collectible-data)
-       :description   (:name collection-data)
        :theme         theme}]
      [reanimated/scroll-view
       {:style     (style/scroll-view top)
        :on-scroll #(on-scroll % scroll-amount title-opacity title-bottom-coord)}
       [collectible-details
-       {:collectible      collectible
-        :set-title-bottom set-title-bottom
+       {:set-title-bottom set-title-bottom
         :theme            theme}]]]))

